chain/client/src/client_actor.rs

```diff
@@ -1508,6 +1512,38 @@ impl ClientActor {
             },
         );
     }
+
+    /// Periodically compute RocksDB stats.
+    fn compute_rocksdb_stats(&self, ctx: &mut Context<Self>) {
+        if let Some(period) = self.client.config.compute_rocksdb_stats_period {
```

Comment:
It may be a good idea to rename this to add `_trigger_` suffix. This will make easier to distinguish trigger from not trigger functions..